website/content/api-docs/secret/azure.mdx

```diff
@@ -216,6 +216,8 @@ information about roles.
   Accepts time suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine default TTL time.
 - `max_ttl` (`string: ""`) – Specifies the maximum TTL for service principals generated using this role. Accepts time
   suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine max TTL time.
+- `explicit_max_ttl` (`string: ""`) – Specifies the explicit maximum lifetime of the lease and service principal.
+  If not set or set to 0, will use the system default.
```

Comment:
nit: is it worth mentioning what the system default is (10 years I believe)?

Reply:
Good idea. I will update it!